### ISSUE TITLE
pgf backend includes wrong

### ISSUE THREAD

**Issue by Cadair** [Cadair/texfigure#1]
currently the pgf backend is not importing rasters properly, despite the use of the include tex module.

**Comment by wtbarnes** [comment]
I just came across this issue as well. I just get the name of the png that it renders in place of the figure. Did you ever come up with a solution for this?

**Comment by Cadair** [comment]
I didn't. iirc I spent a while going around in circles as is the way with latex lol.

**Comment by wtbarnes** [comment]
Yeah I've wasted so much time already haha. Latex is such a black hole.
